index.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Create an array with values that are present in the first input array but not additional ones.
+ *
+ * @example
+ *
+ * const arrayDiffer = require('array-differ');
+ *
+ * arrayDiffer([2, 3, 4], [3, 50]);
+ *  //=> [2, 4]
+ *
+ * @param input - The base array for comparision
+ * @param values - Arrays of value to exculed
+ * @returns  A new array with 'values' excluded
+ */
+export default function arrayDiffer(input: (string|number)[], ...values: (string|number)[]): (string|number)[];
```

Comment:
I think you need to read the description of this module and look at the tests on how it works, because this is completely wrong.

You could look at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5565fe5e46e329a5ee02ddf739abe11bf16f278d/types/lodash/common/array.d.ts#L109 for inspiration.